src/Pages/Main/HomeHead.js

```diff
@@ -0,0 +1,274 @@
+import React from 'react';
+import styled from 'styled-components';
+import Slider from 'react-slick';
+import 'slick-carousel/slick/slick.css';
+import 'slick-carousel/slick/slick-theme.css';
+
+const bannerSlide = [
+  { id: 1, url: 'images/Main/slide1.webp' },
+  { id: 2, url: 'images/Main/slide2.jpeg' },
+  { id: 3, url: 'images/Main/slide3.jpeg' },
+  { id: 4, url: 'images/Main/slide4.webp' },
+  { id: 5, url: 'images/Main/slide5.webp' },
+];
```

Comment:
```suggestion
const BANNER_SLIDES = [
  { id: 1, url: 'images/Main/slide1.webp' },
  { id: 2, url: 'images/Main/slide2.jpeg' },
  { id: 3, url: 'images/Main/slide3.jpeg' },
  { id: 4, url: 'images/Main/slide4.webp' },
  { id: 5, url: 'images/Main/slide5.webp' },
];
```
위와 같이 상수 선언시에는 대문자와 snake_case를 사용해주세요! 
배열이라 복수명사 형태로 S 를 붙여주었습니다!

Reply:
컨벤션 까먹고 있었습니다 감사합니다~~